client_test.go

```diff
@@ -86,12 +85,15 @@ func TestClientQuery(t *testing.T) {
 	autopilot.Equals(t, "1234", q.Account.Id)
 }
 
+/*
+These tests don't work very well with our autopilot endpoint stuff because they need to make recursive calls
+We need to figure out a better way to handle mapping different payloads to different requests
+
```

Comment:
I tried writing some tests for the pagination stuff but its not working correctly and i think its because of how i'm mocking out the opslevel API.  I will have to come back later and figure out a better solution for mocking the opslevel API server for recursive pagination calls.